packages/algob/src/lib/script-checkpoints.ts

```diff
@@ -28,7 +28,7 @@ export function toCheckpointFileName (scriptName: string): string {
 }
 
 export function toScriptFileName (filename: string): string {
-  filename = filename.replace(artifactsPath + path.sep, '');
+  filename = filename.replace(artifactsPath + String(path.sep), '');
```

Comment:
why do we need to add `String`?

Reply:
i don't know, linter was angry so i had to add it